modules/core/03-connection/genesis.go

```diff
@@ -18,6 +20,9 @@ func InitGenesis(ctx sdk.Context, k keeper.Keeper, gs types.GenesisState) {
 		k.SetClientConnectionPaths(ctx, connPaths.ClientId, connPaths.Paths)
 	}
 	k.SetNextConnectionSequence(ctx, gs.NextConnectionSequence)
+	if err := gs.Params.Validate(); err != nil {
```

Comment:
Do we really need to validate here if validation of the params is already done in [`Validate` of 03-connection `GenesisState`](https://github.com/cosmos/ibc-go/blob/main/modules/core/03-connection/types/genesis.go#L77)?